lib/cloudant.js

```diff
@@ -900,6 +887,13 @@ Cloudant.prototype.autoupdate = function(models, cb) {
 
 /**
  * Perform automigrate for the given models.
+ * Notes: Cloudant stores a Model as a design doc in database,
+ * `ddoc` is the doc's _id, auto-generated as 'lb-index-ddoc' + modelName,
+ * `name` is the index name, auto-generated as 'lb-index' + modelName.
+ * It does NOT store properties in the doc, which makes autoupdate actually
+ * does the same thing as automigrate.
+ * It does NOT destroy previous model instances if model exists, user needs
+ * to manually destroy them if necessary.
  *
  * @param {String[]} [models] A model name or an array of model names. If not
```

Comment:
While you're here, since you've updated the JSDoc description, can you also change the parameter type to be `{String|String[]}`? The description to the right of the parameter definition doesn't match the given type definition.